phpunit.xml

```diff
@@ -9,5 +9,6 @@
     <php>
         <server name="KERNEL_DIR" value="/test/app/" />
         <server name="OPEN_ERROR_IN_BROWSER" value="true" />
+        <server name="SUPPORT_DOCTRINE_ORM" value="true" />
```

Comment:
Use? Support does not sound right in that context. The library and tests support doctrine orm no matter if you use it or not.